### PR TITLE
Joint trajectory server race fix

### PR DESCRIPTION
**Pull request by hello-atharva**
## Summary

Fixes abort state inside FollowJointTrajectory action. To test the new functionality, xterm is now used to teleoperate the robot.

## How to test

Build the colcon workspace and launch keyboard teleop:
```
ros2 launch stretch_core keyboard_teleop.launch.py 
```

You should now be able to quickly send out joint trajectory goals without any deadlocks or state transition warnings.

### CODE REVIEW

File: stretch_moveit2/stretch_moveit_config/src/moveit_draw.cpp [hunk 1]
```diff
@@ -140,17 +140,17 @@ int main(int argc, char** argv)
     RCLCPP_INFO(LOGGER, "Visualizing plan (joint space goal) %s", success ? "" : "FAILED");
 
     if(i <= 1){
-      disp_plan.trajectory = my_plan.trajectory;
+      disp_plan.trajectory_ = my_plan.trajectory_;
```
Comment (hello-chintan) [inline stretch_moveit2/stretch_moveit_config/src/moveit_draw.cpp:143]: Moveit related changes are unrelated to this PR and are probably not necessary. Consider raising a separate PR?

**Comment by hello-chintan** [comment]
@hello-atharva Thanks for raising this PR. It seems there are a few unrelated changes in this PR:
1. Fix for executing keyboard teleop node from launch file
2. Fix for race condition in joint trajectory server
3. Fix for MoveIt 2 API changes

Would you consider raising three separate focused PRs with above titles instead of bundling them up together?

**Comment by hello-atharva** [comment]
@hello-chintan 

WIll do! We can remove 1 and 3 from the diff and move those to separate PRs.
